src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/PingGHEventSubscriber.java

```diff
@@ -52,8 +53,18 @@ protected boolean isApplicable(Job<?, ?> project) {
      */
     @Override
     protected void onEvent(GHEvent event, String payload) {
-        // something like <https://github.com/bar/foo>
-        String repo = fromObject(payload).getJSONObject("repository").getString("url");
-        LOGGER.info("{} webhook received from repo <{}>!", event, repo);
+        JSONObject parsedPayload = fromObject(payload);
+        JSONObject repository = parsedPayload.optJSONObject("repository");
+        if (repository != null) {
+            // something like <https://github.com/bar/foo>
```

Comment:
Not sure it matters, but not that this is not actually the form of the URL; it is something involving `api.github.com`.